query-languages/m/m-spec-consolidated-grammar.md

```diff
@@ -384,9 +384,6 @@ _field-list:<br/>
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;field_  `,`  _field-list<br/>
 field:<br/>
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;field-name_  `=`  _expression<br/>
-field-name:<br/>
```

Comment:
It seems like it might make more sense to leave this one, and remove the one below.

Reply:
Sounds good. Swapped.